examples/alphaTestExamples/MultipleServices.md

```diff
@@ -57,8 +57,8 @@ Use the kapply init command to generate the inventory template. This contains
 the namespace and inventory id used by apply to create inventory objects.
 <!-- @createInventoryTemplate @testE2EAgainstLatestRelease-->
 ```
-kapply init $BASE/mysql
-kapply init $BASE/wordpress
+kapply init $BASE/mysql --inventory-namespace default
```

Comment:
This slightly worries me. Do we have to put this here? If we don't use this flag, don't we use the default namespace as the inventory namespace? Don't we get the same result without adding this?

Reply:
removed.

Reply:
So this is a pretty significant change. This means that for every resource that doesn't have a namespace set, we assume that it will end up in the default namespace. I don't think that is a good assumption:

* It might be a cluster-scoped resource.
* Users might have changed the default namespace to something other than `default` in their kubectl context.
* Users might use the `--namespace` flag during apply to put resource without namespace set into another namespace than `default`.

Because of this, I think we shouldn't make any assumptions about the namespace unless it is actually specified in the resource.

WDYT? @seans3

Reply:
Agree that apply everything in default silently isn't good. I would assume there are also a lot of cases that user doesn't put any namespace for all the resources (especially running demos). It would be nice to just run and work, instead of trying again with flag. How about return string saying "inventory initiated in default namespace" or alike.  To make user aware of that.

Reply:
@seans3 @mortent

Reply:
Yeah, I think I agree with your suggestion. Using the `default` namespace if we can't find another one is ok. But lets make sure we include this information in the output so the user can immediately tell which namespace was chosen.

Reply:
updated the pr, @seans3 @mortent  please take a look.